native/cocos/bindings/jswrapper/MappingUtils.h

```diff
@@ -36,32 +37,101 @@ class Object;
 class NativePtrToObjectMap {
 public:
     // key: native ptr, value: se::Object
-    using Map = ccstd::unordered_map<void *, Object *>;
+    using Map = ccstd::unordered_multimap<void *, Object *>;
 
     static bool init();
     static void destroy();
-
     static bool isValid();
 
-    static Map::iterator find(void *nativeObj);
+    CC_DEPRECATED(3.7)
+    static Map::iterator find(void *v);
+    CC_DEPRECATED(3.7)
+    static Map::iterator begin();
+    CC_DEPRECATED(3.7)
+    static Map::iterator end();
```

Comment:
Should describe what functions to use if they are deprecated.

Reply:
Added